docs/groovy/conceptual/query-engine/parallelization.md

```diff
@@ -1,152 +1,540 @@
 ---
-title: Parallelizing queries
+title: Parallelization
 sidebar_label: Parallelization
 ---
 
-Deephaven supports using multiple processors to speed up query evaluation. The extent to which Deephaven employs
-multiple processors depends on both the phase of operation and the query itself.
+Deephaven parallelizes queries automatically, using multiple CPU cores to improve data processing performance. Understanding how parallelization works is essential for writing efficient queries and ensuring correct results.
 
-## Query initialization and updates
+> [!IMPORTANT] > **Breaking change in Deephaven 0.41+**: Queries now run in parallel by default. Non-thread-safe code will produce incorrect results.
```

Comment:
You can't make sweeping generalizations like "queries".  You need to list specific operations (select, update, where).  It will take time to actually review this carefully.